examples/stable_diffusion_v2/ldm/data/dataset.py

```diff
@@ -253,6 +258,18 @@ def tokenize(self, text):
 
         return result
 
+    def _clip_tokenize(self, texts):
+        batch_encoding = self.tokenizer(
+            texts,
+            truncation=True,
+            max_length=self.tokenizer.context_length,
+            return_length=True,
+            return_overflowing_tokens=False,
+            padding="max_length",
+        )
+        tokens = np.array(batch_encoding["input_ids"], dtype=np.int32)
```

Comment:
I think the dtype should be np.int64?

Reply:
It is following the implementation in SDXL